app/experimenter/nimbus-ui/visualization/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@experimenter/visualization",
```

Comment:
So, there is a good chance we won't end up using this visualization package so I don't think we need to copy it over. I think we will end up using another graphing library instead of doing all the svelte shenanigans

Reply:
Sweet! I think after one more ticket we'll be safe to completely delete the `rapid` project under `nimbus-ui` - looks like this package currently lives at `legacy-ui/visualization` and not inside `rapid` so it can be kept around where it is now.

Reply:
Ok this is my bad because I didn't add other tickets for 2 more PRs that closed under legacy for visualization. I will add them